apps/web/src/components/workflow/FlowEditor.tsx

```diff
@@ -102,6 +102,7 @@ export function FlowEditor({
     setTimeout(() => {
       initializeWorkflowTree();
     }, 0);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
some hooks require more careful refactoring, so I decided to leave this comment for now